### PR TITLE
Make address format compatible with other fs.

### LINKED ISSUE #4: Address compatible with other fs.
In OSS, we didn't use an address like `oss://bucket/path` like in S3 or hdfs.
Instead, endpoints are always required for an OSS address.

Currently, the ossfs can only recognize an address like `http://oss-cn-hangzhou.aliyuncs.com/mybucket/myobj`, while the input might be `oss://bucket/path`

### PR DESCRIPTION
**Pull request by karajan1001**
fix #4

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 setup(
     name="ossfs",
-    version="0.1",
+    version="20210609",
```
Comment (efiop) [inline setup.py:12]: What happened to the version format? Seems like the dots are missing

Comment (efiop) [reply setup.py:12]: I also see that this was released on pypi already, you might have to remove that in `https://pypi.org/manage/project/ossfs/releases/`

Comment (karajan1001) [reply setup.py:12]: @efiop , Yes, the version was forgotten to change last time. So what should I do next? First submit a release without version then change to the new format?

Comment (efiop) [reply setup.py:12]: @karajan1001 Just delete the `20210609` version from pypi through that link above and then publish a correct one.

**Comment by efiop** [comment]
@karajan1001 Looks like we are getting timeout errors:

```
2021-06-15T08:09:39.6091343Z     def readinto(self, b):
2021-06-15T08:09:39.6092191Z         """Read up to len(b) bytes into the writable buffer *b* and return
2021-06-15T08:09:39.6093191Z         the number of bytes read.  If the socket is non-blocking and no bytes
2021-06-15T08:09:39.6094036Z         are available, None is returned.
2021-06-15T08:09:39.6094669Z     
2021-06-15T08:09:39.6095355Z         If *b* is non-empty, a 0 return value indicates that the connection
2021-06-15T08:09:39.6096246Z         was shutdown at the other end.
2021-06-15T08:09:39.6096729Z         """
2021-06-15T08:09:39.6097249Z         self._checkClosed()
2021-06-15T08:09:39.6097933Z         self._checkReadable()
2021-06-15T08:09:39.6098569Z         if self._timeout_occurred:
2021-06-15T08:09:39.6099278Z             raise OSError("cannot read from timed out object")
2021-06-15T08:09:39.6099910Z         while True:
2021-06-15T08:09:39.6100383Z             try:
2021-06-15T08:09:39.6100929Z >               return self._sock.recv_into(b)
2021-06-15T08:09:39.6101543Z E               socket.timeout: timed out
```

is there a way to increase the timeouts for ossfs? How does s3fs and others deal with that?

**Pull request merged**